.env

```diff
@@ -69,6 +69,9 @@ CALDAV_ENABLED=true
 CARDDAV_ENABLED=true
 WEBDAV_ENABLED=false
 
+# Can calendar be public available ?
+PUBLIC_CALENDAR_ENABLED=false
+
```

Comment:
Yes, you need to put that in .env.test, and also in the Docker env. But if you give a default value in services, it should be good